docs/dev/Profiling.md

```diff
@@ -0,0 +1,68 @@
+# Profiling Lazygit
+
+If you want to investigate what's contributing to CPU or memory usage, start
+lazygit with the `-profile` command line flag. This tells it to start an
+integrated web server that listens for profiling requests.
+
+## Save profile data
+
+### CPU
+
+This command performs a CPU profile and saves it to a file:
```

Comment:
To be extra clear here I'd say to run this command while lazygit is running with the -profile flag in another terminal session

Reply:
Sure, added in 4546dbe76bfc.